services/core-web/src/components/common/RenderGroupedSelect.js

```diff
@@ -50,10 +51,6 @@ const RenderGroupedSelect = (props) => (
       showSearch
       placeholder={props.placeholder}
       optionFilterProp="children"
-      filterOption={(input, option) =>
```

Comment:
Are these the changes that fixed the issue with the inspector selection? Also is there no filtering needed on this common component for other sections of the application where it's used?

Reply:
Yes. The filtering happens natively so I don't know if this is a workaround that was needed but everything works as expected now.

Reply:
Sounds good!